components/automate-ui/src/app/pages/project/details/project-details.component.html

```diff
@@ -99,10 +99,15 @@
       <ng-container *ngIf="showFirstRuleMessage()">
```

Comment:
Just a thought: might this be more clear?
```suggestion
      <ng-container *ngIf="showNoRulesMessage()">
```